runtime/caml/misc.h

```diff
@@ -539,6 +539,8 @@ extern int caml_snwprintf(wchar_t * buf,
 /* platform dependent best-effort thread naming */
 extern void caml_thread_setname(const char* name);
 
+extern char *caml_strerror(int errnum);
```

Comment:
It don't think this can be `const`, since the pointer returned by the underlying strerror implementation is a `char *` (for historical reasons) and that might point to other storage than our TLS buffer.

Reply:
The prototype of `strerror` itself is a `char*`, so lets keep it that way then.
Although I thought you could always return a `char*` as `const char*`, i.e. advertise the intention to not modify it, but not the other way around (if you were given something with the promise not to modify you can't go ahead and modify it), but its been a while since I've dabbled in C code.